src/features/unified-share-modal/ContactRestrictionNotice.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import * as React from 'react';
+import noop from 'lodash/noop';
+import getProp from 'lodash/get';
+import uniqueId from 'lodash/uniqueId';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import type { InjectIntlProvidedProps } from 'react-intl';
+
+import FormattedCompMessage from '../../components/i18n/FormattedCompMessage';
+import Param from '../../components/i18n/Param';
+import Label from '../../components/label';
+import PlainButton from '../../components/plain-button';
+import InlineNotice from '../../components/inline-notice';
+import LoadingIndicator from '../../components/loading-indicator/LoadingIndicator';
+import SingleSelectField from '../../components/select-field/SingleSelectField';
+
+import ContactsEmailsTooltip from './ContactsEmailsTooltip';
+import messages from './messages';
+
+import type { SelectOptionProp } from '../../components/select-field/props';
+import type { contactType as Contact } from './flowTypes';
+
+import './ContactRestrictionNotice.scss';
+
+type Props = {
+    error?: React.Node,
```

Comment:
This is for accepting any node that can be rendered in React: https://flow.org/en/docs/react/types/#toc-react-node

Most of our UI elements use `React.Node` for errors I think because you can also pass a `<FormattedMessage />` instead of just a string.